src/lru_blockstore.rs

```diff
@@ -17,6 +18,11 @@ impl<const MAX_MULTIHASH_SIZE: usize> LruBlockstore<MAX_MULTIHASH_SIZE> {
             cache: Mutex::new(LruCache::new(capacity)),
         }
     }
+
+    /// Get the number of elements in the blockstore
+    pub fn len(&self) -> Option<usize> {
+        Some(self.cache.lock().ok()?.len())
```

Comment:
Please use `expect` here, the same way we did in the trait methods.

Reply:
Done